salt/metalk8s/kubernetes/cni/calico/configured.sls

```diff
@@ -63,7 +63,7 @@ Create CNI calico configuration file:
             snat: true
             capabilities:
               portMappings: true
-            conditionsV4: ["-d", "{{ grains.metalk8s.workload_plane_ip }}/32"]
+            conditionsV4: ["-d", "{{ grains.metalk8s.workload_plane_ip }}/32,127.0.0.1/32"]
```

Comment:
Shouldn't it be on the whole `127.0.0.0/8` network ?

Reply:
Hmmm I'm not sure, to me here we only want localhost and not any other "loopback" addresses used for .... whatelse
I think but not sure

Reply:
IIUC, what we want is "all host-local addresses except control plane", no?

Reply:
To me, we want workload plane IP and that's it (just we add 127.0.0.1 as it's needed for conformance tests and does not hurt anyway)

Reply:
I don't really know what we should do either and I'm not even sure that's really important.
We can merge your changes as it is what's requested by the K8s conformance tests.

Reply:
Agree :+1:,
Maybe we will want to make it configurable at some point also, but for the moment let's not bother about it :smiley:

Reply:
Likely, yes.

Reply:
Though, maybe the current change is sufficient. It passes conformance, so it's supposed to be conformant. I'm not sure where it's said a K8s cluster *must* expose `NodePort`s on `127.0.0.0/8`. If this is the case, maybe we should widen things up further.